### PR TITLE
Dev modeling users

### PR DESCRIPTION
**Pull request by davitomix**
- Added models user, post and comments.
- Added relations between user, post and models.
- Created users and add posts.
- Created posts and added comments to this posts.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ Style/ClassAndModuleChildren:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
-Layout/HashAlignment:
```
Comment (davitomix) [inline .rubocop.yml]: When run 'rubocop' in local env puts out 'Error: The `Layout/AlignHash` cop has been renamed to `Layout/HashAlignment`.
(obsolete configuration found in .rubocop.yml, please update it)'
but when I do the changes and it gets updated in the repo, the stickler requires the obsolete version of that field.

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  validates :name, presence: true, length: { maximum: 60 }
+  validates :email, presence: true,
+                    format: { with: URI::MailTo::EMAIL_REGEXP },
+                    uniqueness: { case_sensitive: false }
```
Comment (samgaco) [inline app/models/user.rb]: :+1:

File: app/models/post.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  validates :content, presence: true, length: { maximum: 200 }
+  validates :user_id, presence: true
+end
```
Comment (samgaco) [inline app/models/post.rb]: :+1:

File: app/models/comment.rb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  validates :content, presence: true, length: { maximum: 150 }
+  validates :user_id, presence: true
+end
```
Comment (samgaco) [inline app/models/comment.rb]: :+1:

**Pull request merged**